packages/client/src/components/GameEnd/GameEnd.tsx

```diff
@@ -29,9 +27,18 @@ const config: Record<string, Config> = {
   },
 };
 
-const GameEnd: React.FunctionComponent<Props> = ({ result, score }) => {
+const GameEnd: FC = () => {
+  const results = useTypedSelector(state => state.game.results);
+  const { result, score } = results;
```

Comment:
можно было бы на месте разобрать
```suggestion
  const { result, score } = useTypedSelector(state => state.game.results);
```